dashboard/graphql/view.py

```diff
@@ -39,4 +46,16 @@ def get_context(self, request):
             if result:
                 setattr(request, 'canvas_user_id', result.user_id)
 
-        return request
\ No newline at end of file
+        return request
+
+    def execute_graphql_request(self, request, data, query, variables, operation_name, show_graphiql=False):
+        if operation_name == 'Assignment':
```

Comment:
This will log if the Assignment course requests Query, this works for now.
 But I think we need to come up with better supporting the even logging in the long run